test/e2e/common/setup-kubestellar.sh

```diff
@@ -86,27 +86,29 @@ echo "wds1 created."
 
 :
 : -------------------------------------------------------------------------
-: Run OCM transport controller executable
+: Run OCM transport controller in a pod
 :
 cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
 KUBESTELLAR_DIR="$(pwd)"
-## this is a temp solution - run as executble process. this should be replaced to run as pod, ideally using helm
-OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc2"
+OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc4"
 curl -sL https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz | tar xz
 cd ocm-transport-plugin-${OCM_TRANSPORT_PLUGIN_RELEASE}
 OCM_TRANSPORT_PLUGIN_DIR="$(pwd)"
 pwd
 echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
-go mod tidy
```

Comment:
I do not think this PR removes the need for `go mod tidy`.

Reply:
go mod tidy is done internally in “make ko-local-build” command.

Reply:
I do not see that in https://github.com/nirrozenbaum/kubestellar/blob/kflex-config/Makefile

Reply:
see here:
https://github.com/kubestellar/ocm-transport-plugin/blob/main/Makefile#L39

Reply:
Oh, sorry, I was looking at the wrong repo's Makefile.

Reply:
Perhaps the `go mod tidy` needs to precede the `go vet`.

Reply:
I will switch the order in ocm plugin. but that will take place only when we use the next ocm plugin rc

Reply:
I see the change in https://github.com/kubestellar/ocm-transport-plugin/blob/main/Makefile . Until kubestellar can use it, I think we need the `go mod tidy` to remain here.